index.bs

```diff
@@ -3555,13 +3558,27 @@ This [=registration extension=] and [=authentication extension=] allows for a si
 : Authenticator extension input
 :: The client extension input encoded as a CBOR text string (major type 3).
 
+<pre>
+CDDL:
+(
+txAuthSimple: tstr,
+)
+</pre>
+
```

Comment:
tho headed in the correct direction IMO (see https://github.com/w3c/webauthn/issues/725#issuecomment-352139120).  

i am thinking this ought to be spec'd more similarly to how [#sctn-generic-txauth-extension](https://w3c.github.io/webauthn/#sctn-generic-txauth-extension) does it:

```
txAuthSimpleArg = ( tstr )  
```